File: Trimester-1/Python/other/car.py
class PremiumCar:
    
    def __init__(self, make, model, year, distance):
        self.make = make
        self.model = model
        self.year = year
        self.distance = distance

    def __str__(self):
        return self.model
    
    def add_distance(self, distance):
        self.distance += distance
        print(f"car has been driven {self.distance} kms.")

File: Trimester-1/Problem-Solving-Using-C/class-work/2024-08-21/car.c
#include <stdio.h>
#include <string.h>

struct car {
    char make[20];
    char model[20];
    int year;
    float price;
};

void displayCar(struct car c) {
    printf("Make: %s\n", c.make);
    printf("Model: %s\n", c.model);
    printf("Year: %d\n", c.year);
    printf("Price: %.2f\n", c.price);
}

int main() {
    struct car c1;
    strcpy(c1.make, "Toyota");
    strcpy(c1.model, "Corolla");
    c1.year = 2019;
    c1.price = 25000.00;

    displayCar(c1);

    return 0;
}